pandas/core/base.py

```diff
@@ -497,6 +498,19 @@ def searchsorted(self, key, side='left'):
         #### needs tests/doc-string
         return self.values.searchsorted(key, side=side)
 
+    # string methods
+    def _make_str_accessor(self):
+        if not com.is_object_dtype(self.dtype):
```

Comment:
Can you add a check here to exclude string methods on MultiIndex objects, too? They have object dtype, too.